testing/bubbleSort.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai'
+import bubbleSort from '../functions/bubbleSort.js'
+
+describe('#bubbleSort', () => {
+  context('takes an array of unsorted integers', () => {
```

Comment:
Do you need `context` here? You're not setting up a specific set of tests that rely on a state or context to run in.